### PR TITLE
Move scripts used in CI to the new location

### PR DESCRIPTION
**Pull request by sergejparity**
This PR moves CI related scripts and helper files to the new location ./scripts/ci.
We are performing this to unify structure of all paritytech repositories. Target is to improve management of the CI related stuff.
And little housekeeping of the .gitlab-ci.yml.
Related issue: https://github.com/paritytech/ci_cd/issues/313
Also this is reflected in article: Policies and Processes: [Where do I start my new project?](https://github.com/paritytech/ci_cd/wiki/Policies-and-Processes:-Where-do-I-start-my-new-project%3F)

### CODE REVIEW

**Review by alvicsam — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # CI
 /.github/ @paritytech/ci @pepoviola
```
Comment (pepoviola) [inline .github/CODEOWNERS:2]: Is this group exists?

```
Unknown owner on line 2: make sure the team @paritytech/ci exists and has write access to the repository
/.github/ @paritytech/ci @pepoviola
```

Comment (pepoviola) [reply .github/CODEOWNERS:2]: ping @sergejparity

Comment (alvicsam) [reply .github/CODEOWNERS:2]: The group exists but should be configured in the repo settings (Settings -> Collaborators and teams -> Add teams -> @ci with write permissions). From our team only @TriplEight has access

Comment (pepoviola) [reply .github/CODEOWNERS:2]: Great, Thanks!

**Review by pepoviola — APPROVED** [review]

**Comment by pepoviola** [comment]
cc @emamihe merging.

**Pull request merged**